jobs/build-webapp.groovy

```diff
@@ -76,11 +77,10 @@ new Setup(steps
     """<p>A comma-separated list of services we wish to deploy (see below for
 options), or the special value "auto", which says to choose the services to
 deploy automatically based on what files have changed.  For example, you might
-specify "dynamic,static" to force a full deploy to GAE and GCS.</p>
+specify "users,static" to force a full deploy to the users service and GCS.</p>
```

Comment:
I think we'll want to make the same changes to the deploy-znd services parameter as well.